source/slang/slang-capability.cpp

```diff
@@ -201,10 +201,12 @@ void CapabilitySet::addPermutationsOfConjunctionForEachInContainer(CapabilityAto
 
         if constexpr (keyholeAtomToPermuteWith == CapabilityName::target)
         {
+            SLANG_UNUSED(knownTargetAtom);
```

Comment:
You can move this to line 194 at the beginning of the function and remove the other line